src/services/formatting/smartIndenter.ts

```diff
@@ -174,22 +179,24 @@ namespace ts.formatting {
                     indentationDelta += options.indentSize;
                 }
 
+                // Update `current` and `parent`.

```

Comment:
nit: This is kind of a 'duh' comment. You can see they are both being assigned to in the following lines.

Reply:
removed